shadows/framework/src/main/java/org/robolectric/shadows/ShadowWebView.java

```diff
@@ -72,6 +86,111 @@
   private boolean canGoBackIsSet;
   private PageLoadType pageLoadType = PageLoadType.UNDEFINED;
   private HitTestResult hitTestResult = new HitTestResult();
+  private static final WebViewFactoryProvider WEB_VIEW_FACTORY_PROVIDER =
```

Comment:
Can this use ReflectionHelpers.createNullProxy(WebViewFactoryProvider.class)?

Reply:
There is an interface instance called `State`, so I use `createDeepProxy` to avoid NPE problem.